.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: ruby
+rvm:
+  - ruby-head
+  - 2.2
+  - 2.1
+  - 2.0.0
+  - 1.9.3
+branches:
+  only:
+    - master
```

Comment:
I just saw your first comment saying that it is connected.
I'll merge and next PR we'll see if it really builds. If not, we'll remove restriction on branch.